Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:5-slim
+
+RUN apt-get update
+RUN apt-get install libpng12-0
+
+ADD package.json /tmp/package.json
+RUN cd /tmp && npm install
+RUN mkdir -p /reactapp && cp -a /tmp/node_modules /reactapp
```

Comment:
This will bloat the image as it will have two copies of `node_modules` - over 100MB each!

Instead, try something like this:
```
WORKDIR /app
ADD package.json /app
RUN npm install
ADD . /app
```

The `ADD` command won't overwrite `node_modules` because it's listed in `.dockerignore`